diff.py

```diff
@@ -2110,7 +2110,7 @@ def field_matches_any_symbol(field: str, arch: ArchSettings) -> bool:
         return re.fullmatch((r"^@\d+$"), field) is not None
 
     if arch.name == "mips":
-        return field[0] == "."
+        return "." in field
```

Comment:
```suggestion
        return "(." in field
```
(not that it matters hugely)

Reply:
leaving how it is, per discord